<launch>
    <node pkg="baseline_navi" type="stage_switch.py" name="stage_switch" output="screen"/>
</launch>